src/lib/ggpo/main.cpp

```diff
@@ -11,12 +11,18 @@
 #include "backends/spectator.h"
 #include "ggponet.h"
 
+#ifdef _WIN32
```

Comment:
Can we move these functions into the platform specific files instead (e.g. `DllMain` goes into `platform_windows.cpp`)?